demos/tableOfContents/index.html

```diff
@@ -39,68 +39,85 @@
             <nav class="row">
                 <div class="flc-toc-tocContainer demo-toc large-4 small-centered columns"></div>
             </nav>
-
-            <h1>Amphibians</h1>
+            <h1>Species</h1>
+            <h2>Amphibians</h2>
```

Comment:
Why were all of these headings moved down one level in index, and why didn't this affect any test output?